### ISSUE TITLE
Harvest data allocation by county

### ISSUE THREAD

**Issue by HeatherMcW** [Huh/PopR_SDGFP#76]
Josh,

Will it be a problem if there are multiple harvest entries for the same year/season/user group?

I broke out the unregulated seasons to account for the different units each county falls under.
For example:
 2005 21B ARD Archery 03: 20 0 10 0 5 0 0 0
 2005 21B ARD Archery 03: 45 0 20 0 3 0 0 0
 2005 21B ARD Archery 03: 1 0 0 0 0 0 0 0

If this is a problem I can sum up all of the parts into a single record.

**Comment by Huh** [comment]
I would generally feel best about a single record.  I know we talked about this a bit before and I handled this by taking the sum in cases where multiple units were being lumped and the max otherwise to avoid duplicate entries.  
All of the units are cross-referenced to the 2014 unit descriptions before use, but this does not take care of fractional overlap.  The cross referencing was mainly to capture the aggregation of units through time and to force certain areas to be a single entity, like the black hills and the *L units. Josh Nowak 


    On Tuesday, December 8, 2015 1:46 PM, HeatherMcW <notifications@github.com> wrote:
 

 Josh,Will it be a problem if there are multiple harvest entries for the same year/season/user group?I broke out the unregulated seasons to account for the different units each county falls under.
For example:
 2005 21B ARD Archery 03: 20 0 10 0 5 0 0 0
 2005 21B ARD Archery 03: 45 0 20 0 3 0 0 0
 2005 21B ARD Archery 03: 1 0 0 0 0 0 0 0If this is a problem I can sum up all of the parts into a single record.—
